vulnerabilities/importers/redhat.py

```diff
@@ -75,21 +78,53 @@ def to_advisory(advisory_data):
     references = []
     if advisory_data.get("bugzilla"):
         bugzilla = advisory_data.get("bugzilla")
+        url = "https://bugzilla.redhat.com/show_bug.cgi?id={}".format(bugzilla)
+        bugzilla_data = requests.get(f"https://bugzilla.redhat.com/rest/bug/{bugzilla}").json()
+        bugzilla_severity_val = bugzilla_data["bugs"][0]["severity"]
+        bugzilla_severity = VulnerabilitySeverity(
+            system=scoring_systems["rhbs"],
+            value=bugzilla_severity_val,
+        )
+
         references.append(
             Reference(
-                url="https://bugzilla.redhat.com/show_bug.cgi?id={}".format(bugzilla),
+                severities=[bugzilla_severity],
+                url=url,
                 reference_id=bugzilla,
             )
         )
 
-    for rhsa in advisory_data["advisories"]:
-        references.append(
-            Reference(
-                url="https://access.redhat.com/errata/{}".format(rhsa), reference_id=rhsa,
+    for rh_adv in advisory_data["advisories"]:
+        # RH provides 3 types of advisories RHSA, RHBA, RHEA. Only RHSA's contain severity score.
+        # See https://access.redhat.com/articles/2130961 for more details.
+
+        if "RHSA" in rh_adv:
```

Comment:
Are you positive 100% this is ALWAYS uppercase?
A case-insensitive check is usually safer.

Reply:
Done